Remotion/Web/Core/Res/Themes/NovaViso/HTML/TreeView.css

```diff
@@ -1,570 +1,588 @@
-/* Style applied to the TreeView. */
-
 ul.treeViewRoot
 {
+  --treeview-chevron-size: 1em;
+  --treeview-chevron-gap: 0.5em;
+  --treeview-padding: calc(var(--treeview-chevron-size) + var(--treeview-chevron-gap));
+
+  box-sizing: border-box;
```

Comment:
shouldn't we do border-box for the other tree elements, too? Or, otherwise, drop it? (am not sure why it's needed in tree.)